integration/README.md

```diff
@@ -0,0 +1,54 @@
+# Integration Tests
+
+This tests core use cases for u-root such as:
+
+- retrieving and kexec'ing a Linux kernel,
+- uinit (user init), and
+- running unit tests requiring root privileges.
+
+## Usage
+
+Run the tests with:
+
+    go test
```

Comment:
or
UROOT_QEMU=path/to/qemu UROOT_KERNEL=path/to/kernel go test

Reply:
that works as well. However, using this command line, you would have to keep appending UROOT_QEMU and UROOT_KERNEL to go test every single time you want to go test using integration, correct?